splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -1622,7 +1622,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "enum": ["edit", "delete", "clone", "enable"]
+            "enum": ["edit", "delete", "clone", "enable", "search"]
```

Comment:
don't forget to update zod schema
https://github.com/splunk/addonfactory-ucc-generator/blob/d0d69ee3d0c71e7e5b64850b67219244b265c031/ui/src/types/globalConfig/pages.ts#L23